src/main/java/com/github/cypher/gui/root/roomcollection/directory/directory.fxml

```diff
@@ -6,7 +6,7 @@
 <?import javafx.scene.layout.AnchorPane?>
 <?import org.kordamp.ikonli.javafx.FontIcon?>
 
-<AnchorPane maxHeight="99999.0" maxWidth="99999.0" minHeight="250.0" minWidth="400.0" xmlns="http://javafx.com/javafx/8.0.111" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.github.cypher.gui.root.roomcollection.directory.DirectoryPresenter">
+<AnchorPane maxHeight="99999.0" maxWidth="99999.0" minHeight="250.0" minWidth="400.0" style="-fx-background-color: -fx-light-base;" xmlns="http://javafx.com/javafx/8.0.111" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.github.cypher.gui.root.roomcollection.directory.DirectoryPresenter">
```

Comment:
Not relevant for this PR as it only handles styling (not resizing) and value wasn't changed.